src/bucket/test/BucketListTests.cpp

```diff
@@ -1414,10 +1415,10 @@ TEST_CASE_VERSIONS("eviction scan", "[bucketlist]")
         });
     };
 
-    SECTION("legacy scan")
-    {
-        test(/*backgroundScan=*/false);
-    }
+    // SECTION("legacy scan")
```

Comment:
If this is dead code just delete it.

Reply:
Garand already made the changes in https://github.com/stellar/stellar-core/pull/4575, I'll just rebase on top when that change lands (I think that's easier)